### PR TITLE
Update links to point to the new documentation.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,12 @@ while parsing.
...
-- [grmtools book](https://softdevteam.github.io/grmtools/master/book/)
-- [lrpar](https://docs.rs/lrpar/)
-- [lrlex](https://docs.rs/lrlex/)
-- [lrtable](https://docs.rs/lrtable/)
+| Latest release                          | master |
+|-----------------------------------------|--------|
+| [grmtools book](https://softdevteam.github.io/grmtools/latest_release/book/) | [grmtools book](https://softdevteam.github.io/grmtools/master/book) |
+| [cfgrammar](https://docs.rs/cfgrammar/) | [cfgrammar](https://softdevteam.github.io/grmtools/master/api/cfgrammar/) |
+| [lrpar](https://docs.rs/lrpar/)         | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrpar/)         |
+| [lrpar](https://docs.rs/lrlex/)         | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrlex/)         |
```
Comment (ptersilie) [inline README.md:145]: s/lrpar/lrlex

File: README.md [hunk 2]
```diff
@@ -137,7 +137,12 @@ while parsing.
...
+| [lrpar](https://docs.rs/lrtable/)       | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrtable/)       |
```
Comment (ptersilie) [inline README.md:146]: s/lrpar/lrtable

File: README.md (hunk 1 above)
Comment (ltratt) [reply README.md]: Fixed in acc1e1d.

File: README.md (hunk 2 above)
Comment (ltratt) [reply README.md]: Fixed in acc1e1d.

File: README.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ while parsing.
 | [grmtools book](https://softdevteam.github.io/grmtools/latest_release/book/) | [grmtools book](https://softdevteam.github.io/grmtools/master/book) |
 | [cfgrammar](https://docs.rs/cfgrammar/) | [cfgrammar](https://softdevteam.github.io/grmtools/master/api/cfgrammar/) |
 | [lrpar](https://docs.rs/lrpar/)         | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrpar/)         |
-| [lrpar](https://docs.rs/lrlex/)         | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrlex/)         |
-| [lrpar](https://docs.rs/lrtable/)       | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrtable/)       |
+| [lrlex](https://docs.rs/lrlex/)         | [lrpar](https://softdevteam.github.io/grmtools/master/api/lrlex/)         |
```
Comment (ptersilie) [inline README.md:145]: Sorry, same for the right columns.

Comment (ltratt) [reply README.md]: Fixed in dbfea99.

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
I'm not quite sure how/why I `bors`d my own PR here -- I think I confused it for something else. Even if @ptersilie retrospectively agrees that the PR should have been merged, I'm very embarrassed that I did this.

**Pull request merged**